tools/ZydisTestEncoderAbsolute.c

```diff
@@ -224,6 +224,7 @@ static ZyanBool RunBranchingTests(void)
     static const ZydisInstructionAttributes prefixes[] = {
         0,
         ZYDIS_ATTRIB_HAS_BRANCH_TAKEN,
+        ZYDIS_ATTRIB_HAS_BND,
     };
     static const char *str_prefixes[] =
```

Comment:
This table has to be updated every time `prefixes` is updated. Suggested name: `PBN` or `PBD`.

Reply:
Ah darn I missed that, I think PBN is a good choice.